src/OrleansRuntime/Core/InsideRuntimeClient.cs

```diff
@@ -414,6 +426,38 @@ public void SniffIncomingMessage(Message message)
             }
         }
 
+        private static void UnwrapCancellationToken(IAddressable target, object arg, InvokeMethodRequest request, int i)
```

Comment:
Can move this method only partially due to `SiloProviderRuntime` through which the grain extension is added to the activation don't exist in `Orleans` assembly, in which the manager is located. Tush move will only worsen the readability.